lib/rspec/core/output_wrapper.rb

```diff
@@ -0,0 +1,26 @@
+module RSpec
+  module Core
+    # Wrapper around an output stream object.
+    class OutputWrapper
+      # @private
+      attr_accessor :output
+
+      # @param output [IO, StringIO] an object to be wrapped
+      def initialize(output)
+        @output = output
+      end
+
+      [:<<, :advise, :autoclose=, :autoclose?, :binmode, :binmode?, :bytes, :chars, :close, :close_on_exec=,
+       :close_on_exec?, :close_read, :close_write, :closed?, :codepoints, :each, :each_byte, :each_char,
+       :each_codepoint, :each_line, :eof, :eof?, :external_encoding, :fcntl, :fdatasync, :fileno, :flush,
+       :fsync, :getbyte, :getc, :gets, :inspect, :internal_encoding, :ioctl, :isatty, :lineno, :lineno=,
+       :lines, :pid, :pos, :pos=, :print, :printf, :putc, :puts, :read, :read_nonblock, :readbyte, :readchar,
+       :readline, :readlines, :readpartial, :reopen, :rewind, :seek, :set_encoding, :stat, :sync, :sync=,
+       :sysread, :sysseek, :syswrite, :tell, :to_i, :to_io, :tty?, :ungetbyte, :ungetc, :write, :write_nonblock].each do |method|
```

Comment:
Sure, i will add a comment to the code. Also this is list of `IO` methods for Ruby >= 1.9.3. And Ruby 1.8.7 `IO` has different set of methods and they are not subset of list above. I noticed Ruby 1.8.7 is currently support, so should i make different methods lists for Ruby >= 1.9.3 and Ruby 1.8.7?